event-land/scss/_variables.scss

```diff
@@ -0,0 +1,17 @@
+/*------------------------------------*\
+  #COLORS
+\*------------------------------------*/
+
+$primary-color: #50cb8b;
+$primary-color-lighter: #effff7;
+$secondary-color: #ffffff;
+$secondary-color-darker: #f1f1f1;
+$tertiary-color: #24222c;
+
+/*------------------------------------*\
+  #FONT
+\*------------------------------------*/
+
+// font family
+$font-family: "Inter", serif;
```

Comment:
The Inter font is a sans-serif font. I noticed that the fallback font is set to "serif." I think it would be better to change it to "sans-serif" for better consistency.